CssSelector and Xpath/CssSelector Project.txt

```diff
@@ -0,0 +1,9 @@
+CSS Selector Project
+
+1. $("[id=userName]")
+2. $("button[class~=btn-primary]")
+3. $("#app > footer > span")
+4. $("#nav > li:nth-child(n+1) > a")
+5. $("option[value='4']")
+6. document.getElementById("sliderValue")
+7. document.querySelector("#userName")
```

Comment:
This gets the element but doesn't set the value
document.getElementsByClassName('range-slider range-slider--primary')[0].value=30